src/privatesend.h

```diff
@@ -206,60 +244,23 @@ class CDarksendBroadcastTx
         READWRITE(sigTime);
     }
 
+    friend bool operator==(const CDarksendBroadcastTx& a, const CDarksendBroadcastTx& b)
+    {
+        return a.tx == b.tx;
+    }
+    friend bool operator!=(const CDarksendBroadcastTx& a, const CDarksendBroadcastTx& b)
+    {
+        return !(a == b);
+    }
+
```

Comment:
Add `explicit operator bool() const` member function to CDarksendBroadcastTx,
return `false` for empty/default-constructed/no tx, else `true` if there is a tx